service/src/main/resources/templates/ehr_extract_template.mustache

```diff
@@ -26,10 +26,12 @@
         <ehrFolder classCode="FOLDER" moodCode="EVN">
             <id root="{{ehrFolderId}}" />
             <statusCode code="COMPLETE" />
+            {{#hasEffectiveTime}}
```

Comment:
According to the MiM ehrFolder.effectiveTime is required

> [1..1] effectiveTime (IVL<TS>)

Reply:
We should include some test cases around the cases where we have ehrComposition which have nullFlavor dates - to show that we can pass the nulllFlavors through to the folder effectiveTime.